tactics/hints.mli

```diff
@@ -56,6 +56,7 @@ module FullHint :
 sig
   type t
   val priority : t -> int
+  val pattern : t -> Pattern.constr_pattern option
```

Comment:
Should we call it `explicit_pattern` to make it clear that this does not behave exactly as the one in 8.16/8.17?

Reply:
I don't think it's necessary. You're probably the only plugin out there that uses it and you're aware of the difference.